nexus/lib/physical_system.py

```diff
@@ -635,6 +635,15 @@ def ae_pp_species(self):
         #end if
         return ae_species,pp_species
     #end def ae_pp_species
+
+    def kf_rpa(self):
+      import numpy as np
```

Comment:
Move this to a top-level import.  Nexus will move to this standard pattern over time, but no need for local import.